### DISCUSSION TITLE
Welcome to Dotnet.Samples.AspNetCore.WebApi Discussions!

### DISCUSSION THREAD

**Discussion by nanotaboada** [nanotaboada/Dotnet.Samples.AspNetCore.WebApi#122] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kdurgaprasad1604** [comment]
I would like to know what is this project is about
